app/services/audit/catalog_to_archive.rb

```diff
@@ -15,6 +15,15 @@ def self.check_child_zip_part_attributes(zmv, results)
         return false # everything else relies on checking parts, nothing left to do
       end
 
+      total_part_size = zmv.total_part_size
+      moab_version_size = zmv.preserved_object.complete_moab&.size || 0
+      if total_part_size < moab_version_size
```

Comment:
And so, if `moab_version_size`'s calculation changes as suggested above, then like the corresponding check when building the zip file from the Moab version, I think we'd want to confirm that `total_part_size > moab_version_size`, instead of less than.  The assumption being that the zips are created without compression (presumably with a bit of additional zip file metadata), and that the (hopefully past 😅) failure mode we're worried about was the silent/uncaught omission of some files when zips were created before that size check was added.

Reply:
@jmartin-sul Maybe I still have this backwards but instead of running with this, I'd like to dig in so I can hopefully get this in my brain better. Here's what I was thinking... let me know where I go off the rails:

* `total_part_size` is the total size of zips on an endpoint for a given druid + version
* `moab_version_size` is the size on disk for a given druid + version (once fixed per your suggestion above)
* the error condition we want to flag now is when we have stuff missing in the cloud
* the way we flag that is by comparing `total_part_size` to `moab_version_size` and if the former is smaller than the latter, then it's likely missing content
* so isn't the conditional for adding a size inconsistency error result when `total_part_size < moab_version_size`?